crates/biome_js_analyze/src/semantic_analyzers/correctness/use_exhaustive_dependencies.rs

```diff
@@ -385,7 +385,7 @@ impl ReactExtensiveDependenciesOptions {
 /// Flags the possible fixes that were found
 pub enum Fix {
     /// When a dependency needs to be added.
-    AddDependency(TextRange, Vec<TextRange>),
+    AddDependency(TextRange, Vec<TextRange>, usize),
```

Comment:
Instead of using tuple, we should certainly switch to a struct to make more sense of these values? What do you think?
We could do the smae for `RemoveDependency`.

Reply:
Yeah, that makes it much clearer